.mergify.yml

```diff
@@ -15,8 +15,11 @@ pull_request_rules:
         type: APPROVE
         message: MickeyMoz 💪
       merge:
-        method: rebase
-        strict: smart
+        # These PRs are just bumping versions, and very unlikely to break
+        # anything. It's not worth the delay of rebasing them
+        # (sometimes multiple times) before merging.
+        method: squash
+        strict: false
```

Comment:
For the bump PRs I guess we can try this. In this case I think we should do the same for the L10N Auto Merge below too.

Reply:
Sure!